_posts/2024-05-24-may-roundup.markdown

```diff
@@ -29,4 +29,15 @@ While Open Source Summit North America was last month, [The New Stack published
 
 ## Presentation: Digging into Valkey
 
-On the subject of Open Source Summit, the talk I gave along side Madelyn Olsen, ["Digging into Valkey" was posted as a video](https://youtu.be/3G6QgwIl-xs).
\ No newline at end of file
+On the subject of Open Source Summit, the talk I gave along side Madelyn Olsen, ["Digging into Valkey" was posted as a video](https://youtu.be/3G6QgwIl-xs).
```

Comment:
```suggestion
On the subject of Open Source Summit, the talk I gave along side Madelyn Olson, ["Digging into Valkey" was posted as a video](https://youtu.be/3G6QgwIl-xs).
```

Reply:
Fixed in my branch. Sorry for making your surname you Danish/Norwegian.